primitives/npos-elections/compact/src/lib.rs

```diff
@@ -239,20 +207,71 @@ fn imports() -> Result<TokenStream2> {
 	}
 }
 
+fn or_invalid_index_impl() -> TokenStream2 {
+	quote! {
+		// Simple Extension trait to easily convert `None` from index closures to `Err`.
```

Comment:
better to have rust documentation in case user build doc for not public item (if I remember it is doable with some argument)

Reply:
also better to have it prefixed with underscore as it is meant to be private for macro usage